discussion_grading/discussion_grading.py

```diff
@@ -2,33 +2,147 @@
 
 from __future__ import annotations
 
+import logging
+from typing import Optional
+
 import pkg_resources
 from django.utils import translation
+from requests.exceptions import ConnectionError as RequestsConnectionError
 from web_fragments.fragment import Fragment
+from xblock.completable import CompletableXBlockMixin
 from xblock.core import XBlock
-from xblock.fields import Scope, String
+from xblock.fields import Float, Integer, Scope, String
 from xblock.utils.resources import ResourceLoader
 from xblock.utils.studio_editable import StudioEditableXBlockMixin
 
-from discussion_grading.utils import _
+from discussion_grading.edxapp_wrapper.comments import get_course_user_stats
+from discussion_grading.edxapp_wrapper.submissions import create_submission, get_score, set_score
+from discussion_grading.enums import DiscussionGradingMethod
+from discussion_grading.utils import _, get_anonymous_user_id, get_username
+
+log = logging.getLogger(__name__)
+loader = ResourceLoader(__name__)
 
+ITEM_TYPE = "discussion_grading"
+MAX_SCORE = 1
+MIN_SCORE = 0
 
-class XBlockDiscussionGrading(StudioEditableXBlockMixin, XBlock):
+
+@XBlock.needs("i18n")
+@XBlock.needs("user")
```

Comment:
I just found out that you can do:
```suggestion
@XBlock.needs("i18n", "user")
```

Reply:
Wow, I didn't know you could do that 
Done: https://github.com/eduNEXT/xblock-discussion-grading/commit/06352b709b38413ad582d918283eb1f9ede244be